src/libs/hooks/__tests__/useShowReview.test.tsx

```diff
@@ -98,4 +114,32 @@ describe('useShowReview', () => {
       expect(mockUpdateInformationWhenReviewHasBeenRequested).not.toHaveBeenCalled()
     })
   })
+
+  describe('FF disableStoreReview', () => {
+    it('should not show the review when we disabled store review', () => {
+      mockIsAvailable.mockReturnValueOnce(true)
+      mockUseReviewInAppInformation.mockReturnValueOnce({ shouldReviewBeRequested: true })
+      mockRequestInAppReview.mockResolvedValueOnce(undefined)
+      mockSettings.disableStoreReview = false
```

Comment:
mon commentaire précédent permet d'éviter de muter une valeur qui pourrait avoir des conséquences sur les autres tests

```suggestion
      mockSettings.mockReturnValueOnce({ data: { disableStoreReview: false } })
```